tests/MongoDB.Bson.Tests/ObjectModel/Decimal128Tests.cs

```diff
@@ -122,6 +124,7 @@ public void ToDecimal_should_throw_when_value_is_out_of_range(string valueString
             exception.Should().BeOfType<OverflowException>();
         }
 
+#if !NETCOREAPP1_0
```

Comment:
Thread.CurrentThread.CurrentCulture is not available in netstandard1.5

Reply:
Fwiw, it's also not available in 1.1, so if we decide to change the framework target for the test project, this ifdef will need to be updated